packages/server/src/main/resources/application.properties

```diff
@@ -8,6 +8,7 @@ spring.datasource.initialize=false
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.flyway.enabled=false
+spring.datasource.hikari.maximum-pool-size=64
```

Comment:
This will be overridden by Orchestration - and I really do think we are doing something very incorrectly - but it is hidden in Hibernatese and thus so difficult to wend the thread

Reply:
Yeah this is just for local dev mainly. The issue we ran into was with workflows that would spawn 100+ individual  requests, a majority of which are async taskrunner requests that each write to the db multiple times. So with only default 10 connections, causes everything to timeout.

Reply:
Each task runner holds a connection for the entire duration!?!?!?!?!!?!?!?!?!?!?!?!?